DOC-10190: improve Verify Install page (#707)
couchbase/docs-sync-gateway: modules/ROOT/pages/get-started-verify-install.adoc

.Preparatory Steps
NOTE: Ensure you have read, and acted-upon, the information and steps in {get-started-prepare--xref} and {get-started-install--xref}

These instructions are for local or server based deployments.

> If you are using a container such as Docker, look at this

Reviewer: ```suggestion
If you are using a container such as Docker, see
```

We can now use the Admin REST API to add a database to our {sgw} cluster.

[NOTE]
====
The `curl` commands on this page requires basic authentication using the `api_admin` {cbs} RBAC user's credentials we created in Step 2 of {get-started-prepare--xref--create-rbac-users}.
You can create the digest by taking a Base64 of `username:password`. For example:

[source,console]
----
DIGEST=`echo -n sync_gateway:password | base64`
echo $DIGEST
# c3luY19nYXRld2F5OnBhc3N3b3Jk

curl --header "Authorization: Basic $DIGEST" ...
----
====

The `curl` command shown in <<ex-add-sgw-db>> below a `traveldb` database pointing to the {cbs}'s `travel-sample` bucket.

[#ex-add-sgw-db]
.Add a Sync Gateway Database
====

> [source]

Reviewer: ```suggestion
[source, bash]
```
